test/extended/topology_manager/resourcealign.go

```diff
@@ -552,6 +557,13 @@ func (pp PodParams) MakeBusyboxPod(namespace, deviceName string) *corev1.Pod {
 			},
 		}
 
+		if cp.Privileged {
+			isTrue := true
+			cnt.SecurityContext = &corev1.SecurityContext{
+				Privileged: &isTrue,
+			}
+		}
```

Comment:
Are you sure you need to run a pod that's fully privileged? This bypasses every baked in security check.

If the issue is that on busybox you now need root, you can use `runAsUser: 0` in the security context. Not awesome, but better than running a privileged container.

Reply:
good point. Let me narrow down the privileges.

Reply:
So `RunAsUser: 0` is not enough, it seems I need at least `CAP_NET_ADMIN` (which is still too much?)
if I removed 'Privileged: true` and I replace with `RunAsUser: 0` I get:
```
Nov 24 17:52:36.585: INFO: Error running /usr/local/bin/oc --namespace=e2e-test-topology-manager-xhjcv --kubeconfig=/home/fromani/clusters/cnflab/kubeconfig rsh -n default -c test-0 test-29pj7 ping -c 3 10.56.217.171:
StdOut>
bind: Permission denied
PING 10.56.217.171 (10.56.217.171): 56 data bytes
command terminated with exit code 2
StdErr>
bind: Permission denied
PING 10.56.217.171 (10.56.217.171): 56 data bytes
command terminated with exit code 2
```